plugins/src/gcc/mod.rs

```diff
@@ -0,0 +1,19 @@
+use gst::glib;
+use gst::prelude::*;
+mod imp;
+
+glib::wrapper! {
+    pub struct BandwidthEstimator(ObjectSubclass<imp::BandwidthEstimator>) @extends gst::Element, gst::Object;
+}
+
+unsafe impl Send for BandwidthEstimator {}
+unsafe impl Sync for BandwidthEstimator {}
```

Comment:
This is not needed anymore with the git version of the glib bindings (can you also remove the others of this?)